blog/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from .views import PostDetailView, PostListView
+
+urlpatterns = [
+    path("", PostListView.as_view(), name="index"),
+    path("post/<int:pk>", PostDetailView.as_view(), name="post-detail"),
```

Comment:
Consider adding a trailing slash to the 'post/<int:pk>' URL pattern for consistency and to follow Django's convention. This can help avoid potential issues with URL matching.